qa/tasks/mgr/test_orchestrator_cli.py

```diff
@@ -3,7 +3,7 @@
 import logging
 from time import sleep
 
-from teuthology.exceptions import CommandFailedError
+from ceph.deployment.drive_group import DriveGroupValidationError
```

Comment:
this is wrong. as, we don't have access to `ceph` python module from the teuthology worker.